docs/install_guides/installing-openvino-apt.md

```diff
@@ -1,164 +1,144 @@
+   ```sh
+   apt-cache search openvino
+   ```   
+   
+
+### Step 2: Install OpenVINO Runtime Using the APT Package Manager
+
+OpenVINO will be installed in: `/opt/intel/openvino_<VERSION>.<UPDATE>.<PATCH>`
+    
+A symlink will be created: `/opt/intel/openvino_<VERSION>`
 
-The following components are installed with the OpenVINO developer package:
+#### To Install the Latest Version
 
-| Component | Description|
-|-----------|------------|
-| [Model Optimizer](../MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md) | This tool imports, converts, and optimizes models that were trained in popular frameworks to a format usable by Intel tools, especially the Inference Engine. <br>Popular frameworks include Caffe\*, TensorFlow\*, MXNet\*, and ONNX\*. |
-| [Inference Engine](../OV_Runtime_UG/Deep_Learning_Inference_Engine_DevGuide.md) | The engine that runs a deep learning model. It includes a set of libraries for an easy inference integration into your applications.|
-| [OpenCV*](https://docs.opencv.org/master/) | OpenCV\* community version compiled for Intel® hardware |
-| [Sample Applications](../OV_Runtime_UG/Samples_Overview.md)           | A set of simple console applications demonstrating how to use the Inference Engine in your applications. |
-| [Demo Applications](@ref omz_demos) | A set of console applications that demonstrate how you can use the Inference Engine in your applications to solve specific use cases. |
-| Additional Tools                                   | A set of tools to work with your models including [Accuracy Checker utility](@ref omz_tools_accuracy_checker), [Post-Training Optimization Tool Guide](@ref pot_README), [Model Downloader](@ref omz_tools_downloader) and other  |
-| [Documentation for Pre-Trained Models ](@ref omz_models_group_intel)                                   | Documentation for the pre-trained models available in the [Open Model Zoo repo](https://github.com/openvinotoolkit/open_model_zoo).  |
-| Deep Learning Streamer (DL Streamer)   | Streaming analytics framework, based on GStreamer\*, for constructing graphs of media analytics components. For the DL Streamer documentation, see [DL Streamer Samples](@ref gst_samples_README), [API Reference](https://openvinotoolkit.github.io/dlstreamer_gst/), [Elements](https://github.com/openvinotoolkit/dlstreamer_gst/wiki/Elements), [Tutorial](https://github.com/openvinotoolkit/dlstreamer_gst/wiki/DL-Streamer-Tutorial). |
+Run the following command:
+```sh
+sudo apt install openvino
+```
 
+#### To Install a Specific Version
 
-## Install Packages
 
-### Set up the OpenVINO™ Toolkit APT Repository
+1.	Get a list of OpenVINO packages available for installation:
+   ```sh
+   sudo apt-cache search openvino
+   ```
+2.	Install a specific version of an OpenVINO package:
+   ```sh
+   sudo apt install openvino-<VERSION>.<UPDATE>.<PATCH>
+   ```
+    For example:
```

Comment:
Looks like a formatting issue. Please check.
![image](https://user-images.githubusercontent.com/3531525/154257825-e787d168-ba23-4589-bb65-bc9ce6556d5c.png)

Reply:
you mean the brackets in orange? I think it's because of the 'sh' tag in the coding block. Does it look incorrect?

Reply:
or you mean the indentations? that looks correct in the output: http://openvino-docs.inn.intel.com/yuan_install_guide_test1/openvino_docs_install_guides_installing_openvino_apt.html

Reply:
`For example:` is marked as a source block. Maybe it is due to GitHub render issues, but it would be great to fix it as well  (indentation?).

Reply:
If you look at the output in the link you can see the formatting is correct. Sometimes it looks incorrect in Github while correct in the final output, and sometimes it's the opposite. Since it's not easy to test the formatting in the final output (using Jenkins takes too long), I think we should keep as it is here. :)